packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTable.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from 'react';
+import { Loader } from '@deriv-com/ui/dist/components/Loader';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { p2p } from '@deriv/api';
+import { Table } from '../../../../components';
+import MyProfileCounterpartiesEmpty from './MyProfileCounterpartiesEmpty';
+import MyProfileCounterpartiesTableRow from './MyProfileCounterpartiesTableRow';
+import './MyProfileCounterpartiesTable.scss';
+
+type TMyProfileCounterpartiesTableProps = {
+    dropdownValue: string;
+    searchValue: string;
+    setShowHeader: (show: boolean) => void;
+};
+
+type TMyProfileCounterpartiesTableRowRendererProps = {
+    id?: string;
+    is_blocked: boolean;
```

Comment:
Camelcase this laterr
```suggestion
    isBlocked: boolean;
```

Reply:
it comes from the actual api response.. there it was is_blocked